scm/driver/bitbucket/git_test.go

```diff
@@ -33,7 +33,10 @@ func TestGitFindCommit(t *testing.T) {
 
 	want := new(scm.Commit)
 	raw, _ := ioutil.ReadFile("testdata/commit.json.golden")
-	json.Unmarshal(raw, &want)
+	err = json.Unmarshal(raw, &want)
```

Comment:
Very weird that this has a `&` since it's using new. Do ur tests fail if u remove the `&`?

Reply:
I think tests run failing, but I don't think they're failing because of this (I changed it & run test again, they're still failing)

![Screenshot from 2022-02-17 20-53-48](https://user-images.githubusercontent.com/55191777/154513623-17e8b7a7-5070-45cb-8094-9e4229959dca.png)